tweetapi/views.py

```diff
@@ -20,37 +19,31 @@ class ProfileView(generics.ListCreateAPIView):
     serializer_class = ProfileSerilizer
 
 class ProfileDetails(generics.RetrieveUpdateDestroyAPIView):
-    authentication_classes = [Authentication]
     permission_classes = [TwitterUserPermission]
     queryset = ProfileModel.objects.all()
     serializer_class = ProfileSerilizer
 
 class TweetView(generics.ListCreateAPIView):
-    authentication_classes = [Authentication]
+    permission_classes = [AllowAny]
```

Comment:
anyone can list and create a tweet?

Reply:
oh didnt pay attention to what i was suppose to do, once authenticated you can perform and view tweet.......my mistake sir thanks for showing me that.